### DISCUSSION TITLE
Request: override/hide method price

### DISCUSSION THREAD

**Discussion by allanemerson** [Shopify/function-examples#421] [Feedback: Delivery Customization API]
We have a shipping method which is configured as free, but in reality this method is for customers with a contracted shipping rate so they pay for shipping via invoice. We use JS in checkout.liquid today to replace "Free" with "Contracted" to not mislead the customer and need a similar way to update the price with an operation. Adding "price" to RenameOperation would take care of this, either by letting us pass a replacement string or empty string to hide it entirely.

**Comment by sabir-nagarro** [comment]
We are having a similiar problem.
